docs/docs/admin/teamforcapella/migration.md

```diff
@@ -36,8 +41,19 @@
        on the old TeamForCapella server.
        ![Disable backup option](../../user/tools/capella/migration/backup-migration.png)
 
-1.  Export the model to the new TeamForCapella server via the Export wizzard
+1.  Export the model to the new TeamForCapella server via the Export wizard
     from TeamForCapella:
     [Export a model to TeamForCapella](../../user/tools/capella/teamforcapella/export/index.md)
 1.  Recreate the backup pipeline:
-    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md)
+    [Set up a TeamForCapella backup](../../user/projects/models/backups/setup.md).
+    Start a pipeline run and verify that the changes are included in the Git
+    repository.
```

Comment:
```suggestion
    Check if any Capella version specific settigs (e. g. CI/CD variables need to be updated). Start a pipeline run and verify that
    1. the changes are included in the Git backup target and make sense
    1. the pipeline on Git passed (ran successfully). If the pipeline failed 
    repository.
```

Reply:
Have my own formulation that I prefer, please check if the new structure also ok for you.